BranchSDK/src/BranchIO/Util/WindowsStorage.cpp

```diff
@@ -132,6 +132,31 @@ WindowsStorage::set(const std::string& key, const std::string& value, Scope scop
     return *this;
 }
 
+bool
+WindowsStorage::getBoolean(const std::string& key, bool& value, Scope scope) const {
+    string registryKey, registryPath;
```

Comment:
Inconsistency. Elsewhere strings are declared as `std::string`

Reply:
There is a lot of inconsistency around the std namespace.  I filed `SDK-639` to follow up on this.

Reply:
👍